.github/workflows/publish.yml

```diff
@@ -12,10 +12,10 @@ jobs:
       - uses: actions/checkout@v2
```

Comment:
i saw there is also a .travis.yml with java 8, maybe we just remove that?

Reply:
Done